justfile

```diff
@@ -53,3 +53,10 @@ render-prettier:
 # Render GitHub issue & PR templates
 render-github:
     just render templates/PULL_REQUEST_TEMPLATE.md.jinja .github/PULL_REQUEST_TEMPLATE.md
+
+
```

Comment:
```suggestion

```

Reply:
I can never remember when and where we use double spaces between new functions, no spaces between new functions (JavaScript *raises fist to sky*) or one :facepalm: Thanks for the edit.